test/models/bot/smooch_3_test.rb

```diff
@@ -326,10 +329,10 @@ def teardown
       end
       pm = ProjectMedia.last
       assert_no_match /#{@media_url}/, pm.text
-      assert_match /#{@media_url_2}/, pm.comments.last.text
+      # assert_match /#{@media_url_2}/, pm.comments.last.text
       assert_equal 'UploadedImage', pm.media.type
-      assert_match /foo/, pm.comments.last.text
-      assert_match /bar/, pm.comments.last.text
+      # assert_match /foo/, pm.comments.last.text
+      # assert_match /bar/, pm.comments.last.text
```

Comment:
Why you commented these lines instead of deleting them?

Reply:
Deleted commented lines